de.create3000.Titania.json

```diff
@@ -0,0 +1,427 @@
+{
+	"app-id": "de.create3000.titania",
+	"branch": "stable",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.32",
+	"sdk": "org.gnome.Sdk",
+	"command": "titania",
+	"finish-args": [
+		"--device=dri",
+		"--filesystem=home",
```

Comment:
what do you need the whole home directory access for? is it possible to use the Filesystem access portal?

Reply:
Titania is an editor, and you should save your files anywhere in the home directory. It is still not yet possible to use Filesystem access portals, because I use Glade to create the user interface, and Glade does not yet support filesystem portals, but it is planed.

Reply:
Huh, I wasn't aware that you couldn't use a [GTKFileChooserNative](https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html) in Glade.

Reply:
There is already a bug report for Titania https://github.com/create3000/titania/issues/31#issuecomment-432574613 that addresses this issue.